Src/AutoFixtureUnitTest/FixtureTest.cs

```diff
@@ -948,8 +948,8 @@ public void InjectCustomUpperLimitWillCauseSutToReturnNumbersInLimit()
             var result = sut.Create<DoublePropertyHolder<int, long>>();
             // Verify outcome
             Assert.True(
-                (result.Property1 >= lower && result.Property1 < upper) &&
-                (result.Property2 >= lower && result.Property2 < upper)
+                (result.Property1 >= lower && result.Property1 <= upper) &&
+                (result.Property2 >= lower && result.Property2 <= upper)

```

Comment:
Ok, let me try to step back a little and recap my understanding.

The limits given to `RandomNumericSequenceGenerator` are "steps" in which define sets from which random number should be drawn. The last limit is the maximum number you can ever draw before the generator is "reset" (and in the following examples, since we start at 1 it'll be the number of draws that can be made).

Examples:
* limits = [1;42] -> first 42 numbers should be unique and between 1 and 42, after that the generator is reset;
* limits = [1;42;80] -> first 42 numbers should be unique and between 1 and 42, then the next 38 numbers will be unique and between 43 and 80; after that the generator is reset;
* limits = [1;42;80;1500] -> first 42 numbers should be unique and between 1 and 42, then the next 38 numbers will be unique and between 43 and 80; then the next 1420 numbers will be unique and between 81 and 1500, after that the generator is reset;
* etc

In all cases, I can draw as many uniques as the given upper limit before the generator is reset (42, 80, 1500).

The current implementation differs in that in all the examples above, the first 41 draws will be between 1 and 41, and the 42th will be 42, then as above.

The test here is correct against the implementation (it checks that it is between 1 and 41) but not against what I believe to be the correct implementation (between 1 and 42). So if #413 if fixed, this test will fail if it stays as is, hence the required modification.